_docs/platforms/dockeree/index.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: guide
+title: StorageOS Docs - DockerEE
+anchor: platforms
+module: platforms/dockeree
+---
+
+# DockerEE
+
+This section of documentation covers the use of StorageOS in Docker Enterprise Edition.
+
+DockerEE and the Universal Control Plane can be executed in different Linux
+distributions. StorageOS supports RHEL, CentOS, Debian, and selected Ubuntu
+images. For more details, check out the supported OS in the [prerequisites page]({%link _docs/prerequisites/systemconfiguration.md %}).
+
+StorageOS only supports Kubernetes nodes managed by Docker Enterprise Edition.
+Therefore, StorageOS volumes can be provisioned and used only by applications
+on those nodes.
+
+> Mixed nodes support is experimental
```

Comment:
I think we should be stronger here. Nodes running a mix of Kubernetes/Docker Swarm workloads are unsupported? I really don't want to get bogged down in endless support calls by people attempting to mount volumes from within Swarm.

Reply:
Done